docs/content/Coding Milestones/PoC2023q1/kubestellar-syncer-subs/kubestellar-syncer-0-deploy-florin.md

```diff
@@ -1,12 +1,12 @@
 <!--kubestellar-syncer-0-deploy-florin-start-->
 Go to inventory management workspace and find the mailbox workspace name.
 ```shell
-kubectl ws root:espw
-pvname=`kubectl get synctargets.edge.kubestellar.io | grep florin | awk '{print $1}'`
-stuid=`kubectl get synctargets.edge.kubestellar.io $pvname -o jsonpath="{.metadata.uid}"`
-kubectl ws root:imw1
-stcid=`kubectl get synctargets.edge.kubestellar.io florin -o jsonpath="{.metadata.annotations['kcp\.io/cluster']}"`
-mbws="$stcid-mb-$stuid"
+espw_space_config="${PWD}/temp-space-config/espw.config"
```

Comment:
(1) By "space_config" you mean what is commonly known as "kubeconfig", right?

(2) I do not see how the "temp-space-config/" here is helpful.

(3) I would make filename the extension ".kubeconfig".